isofit/configs/sections/surface_config.py

```diff
@@ -23,40 +23,51 @@
 import numpy as np
 
 from isofit.configs.base_config import BaseConfigSection
+from isofit.configs.sections.statevector_config import StateVectorConfig
 
 
 class SurfaceConfig(BaseConfigSection):
     """
-    Instrument configuration.
+    Surface configuration.
     """
 
     def __init__(self, sub_configdic: dict = None):
-        self._surface_file_type = str
-        self.surface_file = None
+        self._multi_surface_flag_type = bool
+        self.multi_surface_flag = False
 
         self._surface_category_type = str
         self.surface_category = None
 
-        self._wavelength_file_type = str
-        self.wavelength_file = None
+        self._surface_file_type = str
+        self.surface_file = None
 
-        # Multicomponent Surface
-        self._select_on_init_type = bool
-        self.select_on_init = True
-        """bool: This field, if present and set to true, forces us to use any initialization state and never change.
-        The state is preserved in the geometry object so that this object stays stateless"""
+        self._surface_class_file_type = str
+        self.surface_class_file = None
+
+        self._sub_surface_class_file_type = str
+        self.sub_surface_class_file = None
+
+        self._Surfaces_type = dict
+        self.Surfaces = {}
 
         self._selection_metric_type = str
         self.selection_metric = "Euclidean"
 
-        # Surface Thermal
+        self._select_on_init_type = bool
```

Comment:
Surface-specific parameters are passed into the surface model the same way. Surface models can see the full config so each model and access specific kwargs as needed.